doc/README_DOXYGEN

```diff
@@ -0,0 +1,182 @@
+wolfSSL with Doxygen 1.8.13
```

Comment:
Please run a spell check on this. `comands`, `secions`, `seperated`, `especialy`, `eachother`.